### PR TITLE
Docs: new option -x for gpcheckcat

### PR DESCRIPTION
**Pull request by mperezfuster**
Documents https://github.com/greenplum-db/gpdb/pull/15962 which adds a new option `-x` to `gpcheckcat`.

### CODE REVIEW

File: gpdb-doc/markdown/utility_guide/ref/gpcheckcat.html.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ gpcheckcat [ <options<] [ <dbname>]
      -B <parallel_processes>
      -v
      -A
+     -x "<parameter_name>=value"
```
Comment (lisakowen) [inline gpdb-doc/markdown/utility_guide/ref/gpcheckcat.html.md:25]: should value also be in `< >`?
is it required that the complete option=value be in double quotes?

Comment (mperezfuster) [reply gpdb-doc/markdown/utility_guide/ref/gpcheckcat.html.md:25]: thanks, definitely value should be in <>. From https://github.com/greenplum-db/gpdb/pull/15962 I understand it has to go in double quotes but maybe I am mistaken?

Comment (lisakowen) [reply gpdb-doc/markdown/utility_guide/ref/gpcheckcat.html.md:25]: i'll let engineering weigh in.  i found use of double-quotes to be intersting.  other options use single-quotes.  and values are not allowed to have spaces...

Comment (mperezfuster) [reply gpdb-doc/markdown/utility_guide/ref/gpcheckcat.html.md:25]: @kpraveen457 can you confirm this please?

Comment (nimish350) [reply gpdb-doc/markdown/utility_guide/ref/gpcheckcat.html.md]: It should be -x "<parameter_name=value>". 

Example: -x "disable_cost=3e15"

**Comment by nimish350** [comment]
LGTM +! approved

**Pull request merged**